src/app-common/autocomplete/Autocomplete.js

```diff
@@ -187,7 +187,7 @@ Autocomplete.propTypes = {
   }).isRequired,
   items: PropTypes.arrayOf(PropTypes.shape({
     value: PropTypes.any.isRequired,
-    display: PropTypes.string.isRequired,
+    display: PropTypes.oneOfType( [PropTypes.string, PropTypes.element] ).isRequired,
```

Comment:
Mohit, is this the right syntax to say the display field can be a string or a JSX element?

Reply:
Yea that works, but I updated it to `display: PropTypes.node.isRequired` which means it can be anything (string, number, jsx element, or array)